src/index.ts

```diff
@@ -1,45 +1,41 @@
-import * as rrweb from 'rrweb';
+import { record, EventType } from 'rrweb';
 import * as Sentry from '@sentry/browser';
 import { Dsn, Event } from '@sentry/types';
 
 type RRWebEvent = {
-  type: rrweb.EventType;
+  type: EventType;
   data: {};
   timestamp: number;
   delay?: number;
 };
 
+type RRWebOptions = {
+  [key: string]: any;
+}
```

Comment:
I think you can do something like... `Parameters<typeof record>`

Reply:
oh nice.. tho now i gotta go fix my test :S